FAQ.md

```diff
@@ -85,6 +86,24 @@ Some image `.png` files are grayscale. This indicates that even though these ima
 
 It really depends on what image you're trying to change the colors of, where these colors are specified. Try looking for related files or `.pal` files.
 
+## How do I find absolute ROM offsets?
+When building external tools, it is usually necessary to know where the data you need is stored. 
+
+The first step is to browse the directories of the project and check files that look related to what you're looking for.
+
+Once you've found what you need there, you'll likely end up finding a name in one of the .asm files (example: PokemonPicPointers)
+
+Now it's time to look at the `pokecrystal.map` file. You'll get this file after building the project with `make`. Search the name you found earlier in this file to find a hexadecimal offset. Wait! You're not done yet! 
```

Comment:
The .sym file would be easier to work with for this use case.

To find the absolute offset of a label like `BaseData`, you just find the line for that label in pokecrystal.sym; in this case it's `14:5424 BaseData`. Then calculate `offset = (bank * $4000) + (address - $4000)`; in this case it's `($14 * $4000) + ($5424 - $4000) = $51424`.

(I changed your example because having the address be $4000 is confusing with the constant $4000s.)

Reply:
No problem, just recommending .sym instead of .map since the bank and address are together on the same line. (Also some projects don't output all the symbols in the .map file, only the section names.)

Reply:
Btw, be aware of the [`symbols` branch](https://github.com/pret/pokecrystal/tree/symbols) that makes the .sym and .map files available online without building yourself.